packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -52,12 +54,32 @@ class DatadogTracer extends Tracer {
     }
   }
 
+  startSpan (name, fields) {
+    if (fields && fields.references) {
+      return super.startSpan(name, fields)
+    } else if (fields && fields.childOf) {
+      let parent = fields.childOf
+      if (parent instanceof OTSpan) {
```

Comment:
Why check for an OT span specifically? If an OT span is received that is not ours it won't work.

Reply:
I was under the impression it could be an OT Span. This is now fixed.